net-libs/libiscsi/files/libiscsi-1.18.0-fno-common.patch

```diff
@@ -0,0 +1,11 @@
+--- a/include/iser-private.h
++++ b/include/iser-private.h
+@@ -159,7 +159,7 @@ struct iser_tx_desc {
+ struct iser_cm_hdr {
+ 	uint8_t      flags;
+ 	uint8_t      rsvd[3];
+-} __packed;
```

Comment:
packed attribute? might not be needed, seeing we're only dealing with bytes

Reply:
It was dropped in the upstream patch so I just followed that

Reply:
oh ok, and I think it shouldnt matter